ted_sws/notice_validator/model/coverage_report.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/python3
+
+import abc
+from typing import List, Optional
+
+from ted_sws.core.model import PropertyBaseModel
+
+
+class XPathAssertion(PropertyBaseModel, abc.ABC):
```

Comment:
why this class inherit from abc.ABC?

Reply:
Same, was thinking to make it an interface, but in the end was not needed